### DISCUSSION TITLE
'Excluded' variants

### DISCUSSION THREAD

**Discussion by rmpb94** [PGScatalog/pgsc_calc#97] [Q&A] [answered]
Hi everyone,
I've been trying to use the _pgsc-calc_ pipeline with my data, and for the most part this has been working really well. I just have a bit of an issue with a SNP that has been 'excluded' according to the match data. Is there a way to find out why this SNP has been excluded, and to over-ride this to force it's inclusion?
Thanks,
Róisín

**Comment by smlmbrt** [comment, marked as answer]
Hi @rmpb94,

The variant will likely be excluded for one of the following reasons and one of the following flags/columns will == TRUE (see https://pgsc-calc.readthedocs.io/en/latest/output.html#match for latest documentation):
`column` | Description
-- | --
`is_multiallelic` | True/False flag indicating whether the matched variant is multi-allelic (multiple ALT alleles). Default 
`ambiguous` | True/False flag indicating whether the matched variant is strand-ambiguous (e.g. A/T and C/G variants).
`duplicate_best_match` | True/False flag indicating whether a single scoring file variants has multiple potential matches to the target genome. This usually occurs when the variant has no other_allele, and with variants that have different REF alleles.
`duplicate_ID` | True/False flag indicating whether multiple scoring file variants match a single target ID.

Most of these behaviours can be altered using flags in the command line (https://pgsc-calc.readthedocs.io/en/latest/reference/params.html), for instance `--keep_multiallelic` and` --keep_ambiguous `.

**Reply by rmpb94** [reply]
Great, thanks! I'll investigate and run with the relevant flags. 
Thank you!
